README.md

```diff
@@ -528,7 +533,20 @@ We got autogenerated `MR.images.home_black_18` `ImageResource` in code, that we
 
 You can get images by their name too
 
-in `commonMain` create a `Resources.kt` file with the content below
+in `commonMain` create a `Resources.kt` file with the content below.
+
+The Image generator also supports `svg` files.
+
+If we add to `commonMain/resources/MR/images` files:
+- `car_black.svg`
+
+We got autogenerated `MR.images.car_black` `ImageResource` in code, that we can use:
+
+- Android: `imageView.setImageResource(image.drawableResId)`
+- iOS: `imageView.image = image.toUIImage()`
+
+On Android it is a VectorDrawable,
+on iOS it is UIImage in the Assets catalog with `preserves-vector-representation` set to `true`.
```

Comment:
here should be note that `svg` is supported on `iOS 13+`
https://developer.apple.com/forums/thread/113015